Fasting App/Profile/ViewModel/ProfileSettingsViewModel.swift

```diff
@@ -44,11 +44,11 @@ class ProfileSettingViewModel: NSObject, UIImagePickerControllerDelegate & UINav
         return ProfileSettingsModel(
             profileImage: profileImage,
             name: user?.fullName ?? "",
-            lblAgeValue: String(user?.age ?? 18),
-            lblWeightValue: String(weight ?? "0.0") + (user?.weightUnit ?? "0.0"),
-            lblHeightValue: String(height  ?? "0") + (user?.heightMsureUnit ?? "0"),
-            lblGenderValue: user?.gender ?? "Female",
-            lblActivityValue: user?.activity ?? "Inactive ",
+            age: String(user?.age ?? 18),
+            weight: String(weight ?? "0.0") + (user?.weightUnit ?? "0.0"),
+            height: String(height  ?? "0") + (user?.heightMsureUnit ?? "0"),
```

Comment:
Not sure why you have `String(weight ?? "0.0") + (user?.weightUnit ?? "0.0")`

What is meant to be displayed?